proto/n0core/lib/proto/enums/request_status.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+enum RequestStatus {
+  INITIAL = 0;
```

Comment:
Do you think how to use these statuses, INITIAL and PENDING?

Reply:
Use `INITIAL` on cluster join message or leave message, not meaning something successful or unsuccessful. (Maybe, cluster join message should be send as a Request, though leave message is notification)
I think `PENDING` is the most useful status to notify something creating, modifying, deleting.  It is used like an acknowledgement message that something receive something.

Reply:
1. in `INITIAL`
`INITIAL` is not clearly. I think join message should use `JOIN` and leave message use `LEAVE`. If unknown meaning, successful or failure, the message should use `UNKNOWN`.

How do you think?

2. in `PENDING`

Uh... Did you mean agent receive delete message before complete to create something? Does it happen really? Maybe, DB has the status, so receive error from api to client.

Reply:
I understood it, The join or leave message is defined as a subtype of `Notification`.  So it should be `AgentJoinNotification` or `AgentLeaveNotification` (as I said, join may be request), and it's status should be `INITIAL`.  It's not unknown status.  And it's status comes `SUCCESS` if its joining process completed.  (maybe adding `UNKNOWN` status is required to have nothing to do with this.)

I don't mean agent receive delete message before complete to create something (to delete).
  But it may occur that agent receive delete request that agent creating something other to delete.  `PENDING` is just notify processing some request.